VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/unread-messages-model-mapper.ts

```diff
@@ -0,0 +1,17 @@
+import { UnreadAdminMessageResponse } from 'src/app/services/clients/api-client';
+import { UnreadAdminMessageModel } from '../../waiting-space/models/unread-admin-message-model';
+
+export class UnreadAdminMessageModelMapper {
+    mapUnreadMessageResponseArray(unreadMessagesReponse: UnreadAdminMessageResponse[], conferenceId: string): UnreadAdminMessageModel[] {
+        const unreadMessages: UnreadAdminMessageModel[] = [];
+        unreadMessagesReponse.forEach(x => {
+            const participant = this.mapFromMessageResponse(x, conferenceId);
+            unreadMessages.push(participant);
+        });
+        return unreadMessages;
+    }
+
+    mapFromMessageResponse(unreadMessagesReponse: UnreadAdminMessageResponse, conferenceId: string): UnreadAdminMessageModel {
```

Comment:
Should probably add a test for this as it can be called on its own. Or make it private if it isn't used outside of this class

Reply:
done now => private